src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -1,24 +1,54 @@
 import { Payment_order } from "__generated__/Payment_order.graphql"
 import { BuyNowStepper } from "Apps/Order/Components/BuyNowStepper"
+import { CreditCardInput } from "Apps/Order/Components/CreditCardInput"
 import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
 import { Link } from "Artsy/Router"
 import React, { Component } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { Elements } from "react-stripe-elements"
+import { Collapse } from "Styleguide/Components/Collapse"
 import { Button } from "Styleguide/Elements/Button"
+import { Checkbox } from "Styleguide/Elements/Checkbox"
 import { Flex } from "Styleguide/Elements/Flex"
 import { Col, Row } from "Styleguide/Elements/Grid"
 import { Join } from "Styleguide/Elements/Join"
 import { Spacer } from "Styleguide/Elements/Spacer"
-import { Placeholder } from "Styleguide/Utils/Placeholder"
 import { Responsive } from "Utils/Responsive"
+import { AddressForm } from "../../Components/AddressForm"
 import { Helper } from "../../Components/Helper"
 import { TransactionSummaryFragmentContainer as TransactionSummary } from "../../Components/TransactionSummary"
+import { Address } from "../Shipping"
 
 export interface PaymentProps {
   order: Payment_order
 }
 
-export class PaymentRoute extends Component<PaymentProps> {
+interface PaymentState {
+  country: string
+  hideBillingAddress: boolean
+}
+
+const ContinueButton = ({ order }) => (
```

Comment:
This is OK for now but we might not need this abstraction-- once we're sending mutations, we'll get rid of the `Link` element anyways and likely handle the transition in an `onClick` handler.